src/web/common/components/InfoDialog/InfoDialog.tsx

```diff
@@ -60,27 +61,76 @@ export const InfoDialog = () => {
       <Popper
         open={open}
         anchorEl={anchorEl}
-        modifiers={[{ name: "offset", options: { offset: [0, 12] } }]}
-        // z-index to match the mask's so that popper is above the mask
+        modifiers={[
+          // push Popper down/up by 26px from the anchor
+          { name: "offset", options: { offset: [0, 26] } },
+          // enable arrow positioning, using our arrowRef
+          { name: "arrow", options: { element: arrowRef, padding: 10 } },
+          {
+            name: "preventOverflow",
+            options: {
+              altAxis: true,
+              altBoundary: true,
+              tether: true,
+              rootBoundary: "document",
+              padding: 12,
+            },
+          },
+        ]}
+        // z-index to match the mask's so that Popper is above the mask
         className={
           "z-[99999] max-w-[min(80svw,36rem)] rounded-lg bg-white" +
           (anchorEl ? "" : " !left-1/2 !top-1/2 !-translate-x-1/2 !-translate-y-1/2")
         }
         aria-label="Info Dialog"
       >
-        <div className="flex items-center justify-between rounded-t-lg bg-paperShaded-main px-2 py-1">
-          <InfoTypeIcon color={infoType} fontSize="small" />
+        {({ placement }) => {
+          const isBelow = placement.startsWith("bottom");
+          return (
+            <>
+              {/* functionality to only show the arrow when there's an anchorEl */}
+              {anchorEl && (
+                <span
+                  ref={setArrowRef}
+                  data-popper-arrow
+                  className={`
+                  absolute
+                  size-3
+                  ${isBelow ? "-top-1.5" : "-bottom-1.5"}
+                  before:content-['']
+                  before:absolute
+                  before:w-3
+                  before:h-3
```

Comment:
This can also be simplified to `before:size-3`
```suggestion
                  before:size-3
```